mods/mc_tutorial/init.lua

```diff
@@ -0,0 +1,1708 @@
+                        "image_button[6.6,3.6;0.8,0.8;blank.png;tool_import;<;false;false]",
+                        "tooltip[node_import;Paste selected]",
+                        "tooltip[tool_import;Paste selected]"
+                    }
+                end,
+            },
+            [mc_tutorial.ACTION.PLACE] = {
+                name = "Place node (PLACE)",
+                fs_elem = function()
+                    return {
+                        "field[0.4,2.3;7,0.8;node;Place (node);", context.epop.fields.node or "", "]",
+                        "field_close_on_enter[node;false]",
+                    }
+                end,
+                expanded_elem = function()
+                    return {
+                        "image_button[6.6,2.3;0.8,0.8;blank.png;node_import;<;false;false]",
+                        "tooltip[node_import;Paste selected]",
+                    }
+                end,
+            },
+            [mc_tutorial.ACTION.WIELD] = {
+                name = "Wield item (WIELD)",
+                fs_elem = function()
+                    return {
+                        "field[0.4,2.3;7,0.8;tool;Wield (item);", context.epop.fields.tool or "", "]",
+                        "field_close_on_enter[tool;false]",
+                    }
+                end,
+                expanded_elem = function()
+                    return {
+                        "image_button[6.6,2.3;0.8,0.8;blank.png;tool_import;<;false;false]",
+                        "tooltip[tool_import;Paste selected]"
+                    }
+                end,
+            },
+            [mc_tutorial.ACTION.KEY] = {
+                name = "Press keys (KEY)",
+                fs_elem = function()
+                    local keys = context.epop.fields.key or {}
+                    return {
+                        "textarea[0.4,2.3;7,1.1;;Press (keys);", next(keys) and table.concat(keys, " + ") or minetest.formspec_escape("[none]"), "]",
+                    }
+                end,
+                collapsed_elem = function() 
+                    return {
+                        "label[0.4,3.1;Open the sidebar to modify keys!]"
+                    }
+                end,
+                expanded_elem = function()
+                    return {
+                        "button[0.4,2.9;3.4,0.8;key_add;Add selected]",
+                        "button[4,2.9;3.4,0.8;key_delete;Remove selected]",
+                    }
+                end,
+            },
+            [mc_tutorial.ACTION.LOOK_YAW] = {
+                name = "Look in horizontal direction (LOOK_YAW)",
+                fs_elem = function()
+                    return {
+                        "field[0.4,2.3;7,0.8;yaw;Yaw (horizontal degrees);", context.epop.fields.yaw or "", "]",
+                        "field_close_on_enter[yaw;false]",
+                    }
+                end,
+            },
+            [mc_tutorial.ACTION.LOOK_PITCH] = {
```

Comment:
Have not removed the sidebars for the `LOOK_PITCH`, `LOOK_YAW` and `POS_ABS` events since that would result in formspec size changes without clicking on the expand/collapse button, which may be seen as unexpected behaviour for the formspec.
Have instead added a "no related information" message to sidebar, replacing the item list.

Reply:
These sidecars could be nice to eventually add more detailed instructions.